cli/src/helper.rs

```diff
@@ -22,26 +22,42 @@ pub fn get_password(prompt: &str, confirmation: bool) -> Result<String, Error> {
     Ok(password.interact()?)
 }
 
-pub async fn pick_account(wallet: &Wallet) -> Result<Option<u32>, Error> {
-    let accounts = wallet.get_accounts().await?;
+pub fn get_decision(prompt: &str) -> Result<bool, Error> {
+    let input = Input::<String>::new()
+        .with_prompt(prompt)
+        .default("Yes".into())
+        .interact_text()?;
+
+    match input.as_str() {
```

Comment:
Probably should do `input.to_lowercase()` and only match on lowercase values.

Reply:
yeah, although if someone writes YES or yEs s/he belongs to a prison!

Reply:
Done